CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+# 1.0.0
+
+1.0.0 is a major update of the API. Feature highlights:
```

Comment:
Point out that the API changes are breaking, non-compatible with previous versions.

Feature highlights should go on its own line.

Reply:
Consider using KeepAChangeLog format e.g. https://github.com/jet/Jet.ConfluentKafka.FSharp/blob/v1/CHANGELOG.md (follow link within - its really easy and really useful)

Reply:
thanks @bartelink 
v1.0 is tricky because it's essentially a complete re-write. Following 1.0, this sounds like a great idea.